tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -3904,17 +3928,18 @@ uint32_t SPIRVEmitter::castToInt(const uint32_t fromVal, QualType fromType,
     } else if (isUintOrVecOfUintType(toIntType)) {
       return theBuilder.createUnaryOp(spv::Op::OpConvertFToU, intType, fromVal);
     } else {
-      emitError("unimplemented casting to integer from floating point");
+      emitError("unimplemented casting to integer from floating point", srcLoc);
```

Comment:
casting from floating point to integer unimplemented

Reply:
Done